app/client/templates/elements/balance.js

```diff
@@ -28,13 +28,15 @@ Template['elements_balance'].helpers({
         
         if (EthTools.getUnit() === 'noether') return 'infinite';
 
+        var ethFormat = this.format || '0,0.00';
+
         if(balance){
             if(EthTools.getUnit() === 'usd' || EthTools.getUnit() === 'eur')
                 return EthTools.formatBalance(TemplateVar.get('balance'), '0,0.00');
             else if(EthTools.getUnit() === 'ether')
-                return EthTools.formatBalance(TemplateVar.get('balance'), '0,0.00[0000000000000000]');
+                return EthTools.formatBalance(TemplateVar.get('balance'), ethFormat);
```

Comment:
This was a generic account element and you moved the format now to the accounts page, and on all other places it will simply show now `0,0.00` only. Why?

I agree that passing in a format makes sense, but you shouldn't remove the finney and ether format, if none is given. The point is that this element is usable everywhere with zero configuration and it has its unit formatting build in, youre changes basically completely remove this and make the if checks in line 36 and 38 useless...